core/test/libraries/TickMath.t.sol

```diff
@@ -66,26 +66,89 @@ contract TickMathTest is Test {
         assertEq(TickMath.ceil(-3, 1), -3);
     }
 
-    function test_getTickAtSqrtRatio(uint160 sqrtPriceX96) public {
+    /// forge-config: default.fuzz.runs = 16384
+    function test_comparitive_getTickAtSqrtRatio(uint160 sqrtPriceX96) public {
         if (sqrtPriceX96 < TickMath.MIN_SQRT_RATIO) {
             vm.expectRevert(bytes("R"));
             TickMath.getTickAtSqrtRatio(sqrtPriceX96);
+            vm.expectRevert(bytes("R"));
+            getTickAtSqrtRatioOriginal(sqrtPriceX96);
             sqrtPriceX96 = TickMath.MIN_SQRT_RATIO;
         } else if (sqrtPriceX96 >= TickMath.MAX_SQRT_RATIO) {
             vm.expectRevert(bytes("R"));
             TickMath.getTickAtSqrtRatio(sqrtPriceX96);
+            vm.expectRevert(bytes("R"));
+            getTickAtSqrtRatioOriginal(sqrtPriceX96);
             sqrtPriceX96 = TickMath.MAX_SQRT_RATIO - 1;
         }
 
         int24 a = TickMath.getTickAtSqrtRatio(sqrtPriceX96);
-        int24 b = getTickAtSqrtRatio(sqrtPriceX96);
+        int24 b = getTickAtSqrtRatioOriginal(sqrtPriceX96);
+
+        assertEq(a, b);
+    }
+
+    /// forge-config: default.fuzz.runs = 16384
+    function test_comparitive_getSqrtRatioAtTick(int24 tick) public {
+        if (tick < TickMath.MIN_TICK) {
+            vm.expectRevert(bytes("T"));
+            TickMath.getSqrtRatioAtTick(tick);
+            vm.expectRevert(bytes("T"));
+            getSqrtRatioAtTickOriginal(tick);
+            tick = TickMath.MIN_TICK;
+        } else if (tick > TickMath.MAX_TICK) {
+            vm.expectRevert(bytes("T"));
+            TickMath.getSqrtRatioAtTick(tick);
+            vm.expectRevert(bytes("T"));
+            getSqrtRatioAtTickOriginal(tick);
+            tick = TickMath.MAX_TICK;
+        }
+
+        uint256 a = TickMath.getSqrtRatioAtTick(int24(tick));
+        uint256 b = getSqrtRatioAtTickOriginal(int24(tick));
 
         assertEq(a, b);
     }
 }
```

Comment:
👏 💯 Although your comments on the code for `getSqrtRatioAtTick` were pretty thorough, this test explicitly shows that the two are equal.